master/internal/rm/kubernetesrm/jobs.go

```diff
@@ -394,6 +398,12 @@ func readClientConfig(kubeconfigPath string) (*rest.Config, error) {
 		// and it expects to find files:
 		//   - /var/run/secrets/kubernetes.io/serviceaccount/token
 		//   - /var/run/secrets/kubernetes.io/serviceaccount/ca.crt
+		c, err := rest.InClusterConfig()
+		if err != nil {
+			return nil, err
+		}
+		c.QPS = 100
+		c.Burst = 100
 		return rest.InClusterConfig()
```

Comment:
Shouldn't this `return c, err`?

Reply:
This was ruled out as a viable solution before, I was testing solutions for a working fallback on a diff branch and I think i accidentally made this one here, I'll get rid of this

Reply:
but yea it woudlve had to be `return c, err`